src/plugins/presentation_util/public/components/data_view_picker/data_view_picker.tsx

```diff
@@ -61,58 +61,56 @@ export function DataViewPicker({
   };
 
   return (
-    <>
-      <EuiPopover
-        button={createTrigger()}
-        isOpen={isPopoverOpen}
-        closePopover={() => setPopoverIsOpen(false)}
-        display="block"
-        panelPaddingSize="s"
-        ownFocus
-        panelClassName="presDataViewPicker__panel"
+    <EuiPopover
```

Comment:
All I did here was remove the outer `<> ... </>` to remove an unnecessary layer 😆